apps/pigment-css-vite-app/src/pages/material-ui/react-breadcrumbs.tsx

```diff
@@ -51,7 +73,9 @@ export default function Breadcrumbs() {
       <section>
         <h2> Router Breadcrumbs</h2>
         <div className="demo-container">
-          <RouterBreadcrumbs />
+          <ErrorBoundary FallbackComponent={ErrorBoundaryFallback}>
+            <RouterBreadcrumbs />
```

Comment:
I'll look into the actual fix separately. It's related to nested router usage. So for now, I've wrapped it in `ErrorBoundary` so that rest of the page works.